s3tests_boto3/functional/test_s3select.py

```diff
@@ -112,9 +115,23 @@ def s3select_assert_result(a,b):
     if type(a) == str:
         a_strip = a.strip()
         b_strip = b.strip()
+        if a=="" and b=="":
+            warnings.warn(UserWarning("{}".format("both results are empty, it may indicates a wrong input, please check the test input")))
+            ## print the calling function that created the empty result.
+            stack = traceback.extract_stack(limit=2)
+            formatted_stack = traceback.format_list(stack)[0]
+            warnings.warn(UserWarning("{}".format(formatted_stack)))
+            return a==b
```

Comment:
Could this be replaced with `return True`? It seems that it's been established that `a` and `b` are the same, unless I'm missing something subtle in Python.

Reply:
nothing important.
just explaining this flow, 
assert is TRUE since result-a is equal to result-b.